resources/views/themes/bimbala/blog/post.blade.php

```diff
@@ -11,13 +11,13 @@
     <article id="post-{{ $post->id }}" class="max-w-4xl px-5 mx-auto prose prose-xl lg:prose-2xl lg:px-0">
 
         <meta property="name" content="{{ $post->title }}">
-        <meta property="author" typeof="Person" content="admin">
```

Comment:
What about removing it? We have JSON+LD schema in the `head` which refers the original user as an author. The meta tag here differs from it.